src/fabric/src/fabric2_server.erl

```diff
@@ -137,8 +139,62 @@ init(_) ->
         end,
     application:set_env(fabric, ?FDB_DIRECTORY, Dir),
     config:subscribe_for_changes([?TX_OPTIONS_SECTION]),
+    check_config_limits(),
     {ok, nil}.
 
+check_config_limits() ->
+    ConfigMaxDocSize = config:get_integer(
+        "couchdb",
+        "max_document_size",
+        ?DOCUMENT_SIZE_LIMIT_BYTES
+    ),
+    ConfigMaxAttSize = config:get_integer(
+        "couchdb",
+        "max_attachment_size",
+        ?ATT_SIZE_LIMIT_BYTES
+    ),
+    ConfigDocIdLength = config:get_integer(
+        "couchdb",
+        "max_document_id_length",
+        ?DOC_ID_LIMIT_BYTES
+    ),
+    ConfigBinaryChunkSize = config:get_integer(
+        "fabric",
+        "binary_chunk_sizze",
```

Comment:
sizze -> size

Reply:
The logic has been changed in https://github.com/apache/couchdb/pull/3632/commits/36eb7f16dd8935cd8792a7297aadebe535f109d1.